thirdparty/DiskANN/src/index.cpp

```diff
@@ -1486,7 +1490,12 @@ namespace diskann {
 
       std::vector<std::vector<unsigned>> pruned_list_vector(round_size);
 
-      for (uint32_t sync_num = 0; sync_num < num_syncs; sync_num++) {
+      auto round_num_syncs = num_syncs;
+      if(fast && rnd_no == 0){
```

Comment:
I remember we will reduce the sync size in the second round right?

Reply:
> I remember we will reduce the sync size in the second round right?

no, we use a very small sync size in round one with alpha 1.0. It works like adding multi-eps.

Reply:
Cool, we need more materials in this PR:
1. explanation of how we do this acceleration.
2. test results.

Reply:
I edit the pr comment, adding two documents. It may help better understand this pr.